configs/emcts1/victimplay.cfg

```diff
@@ -0,0 +1,14 @@
+@include ../../engines/KataGo-raw/cpp/configs/training/selfplay1.cfg
```

Comment:
Some other options I remember overriding in our old victimplay config that we may still want to override here:
- forkGameProb. I turned this off because forking games was complicated and not sure how it interacted with our other codepaths. Our new code should be safer though, so worth trying this to see if it improves learning.
- I fixed board size, komi, and ruleset, and handicapProb=0 when attacking the adversary, reasoning being that I thought attacking in as restricted of an environment as possible would be easier / result in weirder attacks.

Overall I think it isn't as important what settings we use during training. We should just do some ablations and use the ones that result in the fastest training / best results in `match`.

Reply:
I would lean towards keeping our victimplay config as close as possible to what it was before and modify it in a new PR once we are sure new victimplay settings still result in a functional attack.

Reply:
> I would lean towards keeping our victimplay config as close as possible to what it was before and modify it in a new PR once we are sure new victimplay settings still result in a functional attack.

:+1: seems like the main thing is to make sure the match configs are OK. We can vary the victimplay configs as much as we want. Certainly worth revisiting them to make sure they're tuned appropriately, but I don't think we should necessarily default to the KataGo defaults.

Reply:
I also quite liked that we made the config hierarchical, was much easier to read/understand.

Reply:
I just undid this